src/api/app/models/status_message.rb

```diff
@@ -31,7 +31,7 @@ def acknowledge!
     users << User.session!
   end
 
-  def self.newest_announcement_for_current_user
+  def self.latest_for_current_user
```

Comment:
You should also rename it where we use it:
spec/models/status_message_spec.rb
app/controllers/webui/webui_controller.rb